iron-overlay-behavior.html

```diff
@@ -72,16 +72,14 @@
       canceled: {
         observer: '_canceledChanged',
         readOnly: true,
-        type: Boolean,
-        value: false
```

Comment:
Could you go into some detail about why we are removing all of the default values?